base_tier_validation/models/tier_review.py

```diff
@@ -77,11 +78,26 @@ def _compute_reviewed_formated_date(self):
 
     @api.depends("definition_id.approve_sequence")
     def _compute_can_review(self):
```

Comment:
@bosd Can you help me understand `can_review` vs. `waiting` and `pending`? Is it always the case that `can_review` is true if and only if its status is `pending`? Would it be safe for me to propose a simplification based on this? Or am I missing something?